fedimint-core/src/db/mod.rs

```diff
@@ -393,7 +393,7 @@ pub trait IDatabaseTransaction<'a>: 'a + MaybeSend {
 /// `ISingleUseDatabaseTransaction` without needing to make additional
 /// allocations.
 #[apply(async_trait_maybe_send!)]
-pub trait ISingleUseDatabaseTransaction<'a>: 'a + Send {
+pub trait ISingleUseDatabaseTransaction<'a>: 'a + MaybeSend {
```

Comment:
Is it possible to write a wasm test so that we catch these issues in CI going forward?

Reply:
Let's make an issue so we don't forget

Reply:
I have mentioned it in #2194